app/gui2/src/components/GraphEditor/widgets/WidgetFileBrowser.vue

```diff
@@ -54,6 +56,23 @@ const label = computed(() => {
 const FILE_CONSTRUCTOR = FILE_TYPE + '.new'
 const FILE_SHORT_CONSTRUCTOR = 'File.new'
 
+const fileConPattern = Pattern.parse(`${FILE_CONSTRUCTOR} __`)
+const fileShortConPattern = Pattern.parse(`${FILE_SHORT_CONSTRUCTOR} __`)
```

Comment:
I think those `Pattern.parse` expressions should simply be part of the consts above. The pure string form is not used by anything else.